### DISCUSSION TITLE
DietPi-Software | Actual Budget

### DISCUSSION THREAD

**Discussion by hello-jason** [MichaIng/DietPi#5492] [Software Request]
#### Formal software information
- Software title | Actual Budget
- Short description | Privacy-focused app for managing your finances
- Official URL (if available) | https://actualbudget.com/
- GitHub URL (if available) | https://github.com/actualbudget/actual
- (Official) install documentation | https://github.com/actualbudget/actual#installation

#### Are there similar/alternative software titles available with DietPi-Software?
- None that I could find

#### How can DietPi make the installation easier or compatible, than following the install instructions or do APT installation, if available?
Looking at the recommendation installation method of using the [actual-server](https://github.com/actualbudget/actual-server) project, it suggests deploying via Docker. DietPI could install the core repo, actual-server, and Docker to save the user time. It could also configure the Docker container to automatically run on bootup.

#### Can you provide the installation steps that you would suggest DietPi-Software to do?
Seems they recommend installing using the [actual-server](https://github.com/actualbudget/actual-server) method.

#### Are you willing to help maintaining the software installation, e.g. in case of needed setup changes due to updates etc.? This is not needed, but could speed up our decision to implement it, as man power is always a topic :wink:.
- I would if I knew how

**Comment by Washstand4019** [comment]
I have have been installing actual budget on my dietpi instance using the docker method.  I would love to see this implemented into dietpi software directly to make it easier to deploy a streamlined architecture version with databases separated from the application for easy backup.  I believe this would be a great addition to the software list.  I would be willing to help contribute here where I can to bring this into main.

**Comment by Ctrl-Alb** [comment]
In the latest update of Actual Budget, [a new installation method has been introduced](https://actualbudget.org/docs/install/cli-tool).
This approach may be more suitable for integration with `DietPi-Software`.
